src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -5299,8 +5299,8 @@ private void MOV_Handler(Instruction instruction)
 
         private void ADD_Handler(Instruction instruction)
```

Comment:
I guess this would mean arithmetic operations should always pop 2 elements off the stack.
Just a thought, it might make sense to continue support for operand based instructions and implement a corresponding instructions that explicitly pop operands off the stack.

add op1, op2, op3 <- current implementation
adds <- instruction that pops from the stack

This way we maintain extensibility from the instruction set.